apps/loan/src/components/PageLlamaMarkets/LendTableTitle.tsx

```diff
@@ -0,0 +1,66 @@
+import Box from '@mui/material/Box'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import LlamaSunglasses from '../../../public/images/llama-sunglasses.png'
+import Image from 'next/image'
+import styled from 'styled-components'
```

Comment:
Why not use `import { styled } from '@mui/material/styles'`, then you can create a styled <img> with sx properties.

Reply:
aha I didn't know why I missed the sx :laughing: